engines/dfc_provider/app/serializers/dfc_provider/catalog_item_serializer.rb

```diff
@@ -13,6 +13,8 @@ class CatalogItemSerializer < ActiveModel::Serializer
              serializer: DfcProvider::OfferSerializer,
              key: 'dfc:offeredThrough'
 
+    delegate :sku, to: :object
```

Comment:
I dont think you need this. AMS will do this for you. I think you can simply remove this line.